app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ErrorsController < ApplicationController
+    def not_found
+        render status: 404
+    end
+
+    def internal_server
+        render status: 500
+    end
+
+    def unprocessable
+        render status: 422
+    end
+
+    def unaccetable
```

Comment:
Also, there's no route or view for this method. So it should be deleted or the appropriate route and view should be added.